training_config/event2mind.json

```diff
@@ -1,29 +1,26 @@
 {
   "dataset_reader": {
     "type": "event2mind",
+    # Uncomment this when generating the vocabularly with `dry-run`.
```

Comment:
@rlebras, you can follow these comments encoded in this config or just use the following commands:
```
        allennlp dry-run -o '{"dataset_reader": {"dummy_instances_for_vocab_generation": true}} {"vocabulary": {"min_count": {"source_tokens": 2}}}' training_config/event2mind.json --serialization-dir vocab_output_path
        allennlp train -o '{"vocabulary": {"directory_path": "vocab_output_path/vocabulary/"}}' training_config/event2mind.json --serialization-dir output_path
```